scripts/combine_metadata.py

```diff
@@ -64,8 +68,8 @@ def parse_args():
     for strain in combined_data:
         for column in combined_columns:
             if column not in combined_data[strain]:
-                combined_data[strain][column] = EMPTY    
-    
+                combined_data[strain][column] = EMPTY
+
     for idx in range(1, len(metadata)):
         for strain, row in metadata[idx]['data'].items():
             if strain not in combined_data:
```

Comment:
_(non-blocking)_

Some of this can probably be done more efficiently if data is kept as a DataFrame, but that should be a separate PR.

Reply:
This script needs to be rewritten to handle larger metadata files at some point, anyway, so I figured pandas optimization could happen there.